.gitignore

```diff
@@ -29,6 +29,7 @@ dlangspec.tex
 
 /web
 /2.*.ddoc
+modlist-*.ddoc
```

Comment:
> This file keeps on growing and I think we should revert that. I think we should make appropriate use of temporary directories. Who generates these files and where should they go at best?

Temp files and dependencies don't work well together, but I moved the files into a `.generated` folder.